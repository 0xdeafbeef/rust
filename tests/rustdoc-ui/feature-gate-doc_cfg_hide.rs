// FIXME: Remove this file once feature is removed

#![doc(cfg_hide(test))] //~ ERROR

#[cfg(not(test))]
pub fn public_fn() {}
#[cfg(test)]
pub fn internal_use_only() {}
