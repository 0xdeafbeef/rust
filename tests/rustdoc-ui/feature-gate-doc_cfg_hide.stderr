error: unknown `doc` attribute `cfg_hide`
  --> $DIR/feature-gate-doc_cfg_hide.rs:3:8
   |
LL | #![doc(cfg_hide(test))]
   |        ^^^^^^^^^^^^^^
   |
   = note: `#[deny(invalid_doc_attributes)]` on by default

error: aborting due to 1 previous error

